root/layout/components/Footer.js

```diff
@@ -1,95 +1,533 @@
+    <button
+      aria-expanded="false"
+      className="btn btn-outline-primary fs-5"
+      data-bs-toggle="dropdown"
+      id="language-dropdown"
+      type="button"
+    >
+      {languageName(
+        serverLanguages.find(x => x.name === currentBCP47Language),
+        true,
+      )}
+    </button>
+    <ul aria-labelledby="language-dropdown" className="dropdown-menu">
+      {serverLanguages.map(function (language, index) {
+        let inner = <LanguageLink $c={$c} language={language} />;
+
+        if (language.name === currentBCP47Language) {
+          inner = <strong>{inner}</strong>;
+        }
+
+        return (
+          <li key={index}>
+            {inner}
+          </li>
+        );
+      })}
+      <li>
+        <a
+          className="dropdown-item"
+          href={'/set-language/unset?' + returnToCurrentPage($c)}
+        >
+          {l('(reset language)')}
+        </a>
+      </li>
+      <li>
+        <a
+          className="dropdown-item"
+          href="https://www.transifex.com/musicbrainz/musicbrainz/"
+        >
+          {l('Help Translate')}
+        </a>
+      </li>
+    </ul>
+  </div>
+);
+
+const Footer = (): React.Element<'section'> => {
   const $c = React.useContext(CatalystContext);
-  const stash = $c.stash;
+  const serverLanguages = $c.stash.server_languages;
+
   return (
-    <div id="footer">
-      <p className="left" id="footer-menu">
-        <a className="internal" href={DONATE_URL}>{l('Donate')}</a>
-        <a className="internal" href="//wiki.musicbrainz.org/">{l('Wiki')}</a>
-        <a className="internal" href="https://community.metabrainz.org/">{l('Forums')}</a>
-        <a className="internal" href="/doc/Communication/IRC">
-          {l('Chat (IRC)')}
-        </a>
-        <a className="internal" href="http://tickets.metabrainz.org/">{l('Bug Tracker')}</a>
-        <a className="internal" href="https://blog.metabrainz.org/">{l('Blog')}</a>
-        <a className="internal" href="https://twitter.com/MusicBrainz">{l('Twitter')}</a>
-
-        {DBDefs.BETA_REDIRECT_HOSTNAME ? (
-          <a
-            className="internal"
-            href={
-              '/set-beta-preference?' + returnToCurrentPage($c)
-            }
-          >
-            {DBDefs.IS_BETA
-              ? l('Stop using beta site')
-              : l('Use beta site')}
-          </a>
-        ) : null}
-      </p>
-
-      <p className="right">
-        {exp.l(
-          `Brought to you by {MeB|MetaBrainz Foundation} and our
-           {spon|sponsors} and {supp|supporters}. Cover Art provided
-           by the {caa|Cover Art Archive}.`,
-          {
-            caa: '//coverartarchive.org/',
-            MeB: 'https://metabrainz.org/',
-            spon: 'https://metabrainz.org/sponsors',
-            supp: 'https://metabrainz.org/supporters',
-          },
-        )}
-
-        {DBDefs.DB_STAGING_SERVER && DBDefs.GIT_BRANCH ? (
-          <>
+    <section className="bs footer">
```

Comment:
@santiagofn here is an example of how we wrap the classes from DS bootstrap and use them in MB